README.md

```diff
@@ -15,40 +15,79 @@ To run this project locally, follow these steps:
 
 Clone the repository to your local machine. Replace `YOUR_USERNAME` with your GitHub username:
 
-git clone https://github.com/YOUR_USERNAME/Nc-news-project.git
+```bash
+git clone https://github.com/YOUR_USERNAME/your-repository-name.git
+```
 
 ### 2. **Create new repository on Git and set a remote branch**
 
 Get a remote branch. Replace `YOUR_NEW_REPO_URL_HERE` with the URL of your new repo in Git.
 
+```bash
 cd path to the clone project
 git init
 git remote add origin YOUR_NEW_REPO_URL_HERE
+```
 
```

Comment:
This step isn't necessary, we are just demonstrating how they can get the repo on their machine and run it locally, these steps suggest we want them to have their own fork.